connection.go

```diff
@@ -31,7 +33,7 @@ func NewConnection(session *Session, localClientID, remoteClientID string) (*Con
 		session:               session,
 		localClientID:         localClientID,
 		remoteClientID:        remoteClientID,
-		windowSize:            uint32(session.config.InitialConnectionWindowSize),
```

Comment:
Remove `InitialConnectionWindowSize` from config since it's no longer used.

Reply:
Yes, removed.